wcivf/apps/elections/tests/test_helpers.py

```diff
@@ -173,7 +173,7 @@ def test_do_import_with_params(self, importer):
         importer.do_import(params={"election_date": "2021-05-06"})
         importer.ee_helper.prewarm_cache.assert_not_called()
 
-        expected_url = f"{settings.YNR_BASE}/api/next/ballots/?election_date=2021-05-06&page_size=200"
+        expected_url = f"{settings.YNR_BASE}/api/next/ballots/?election_date=2021-05-06&page_size=200?{settings.YNR_API_KEY}"
```

Comment:
I think this should be `auth_token={settings.YNR_API_KEY}`

Reply:
is it `&auth_token..` or `?auth_token`?